ovos_plugin_manager/utils/config.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+from ovos_config.config import Configuration
+
+
+def get_plugin_config(config: Optional[dict] = None, section: str = None,
+                      module: Optional[str] = None) -> dict:
+    """
+    Get a configuration dict for the specified plugin
+    @param config: Base configuration to parse, defaults to `Configuration()`
+    @param section: Config section for the plugin (i.e. TTS, STT, language)
+    @param module: Module/plugin to get config for, default reads from config
+    @return: Configuration for the requested module, including `lang` and `module` keys
+    """
+    config = config or Configuration()
+    lang = config.get('lang') or Configuration().get('lang')
+    config = (config.get('intentBox', {}).get(section) or config.get(section)
```

Comment:
should this be behind some other check? will merge for now but we should revisit since this allows any config to live under intentBox section and we likely want to throw an error in that case